buildsystem/spack/crusher/modules/dependencies.sh

```diff
@@ -52,13 +46,19 @@ module load rocrand/5.2.0-clang-14.0.0-rocm5.2.0-mixed-nwexus7
 # rocthrust@=5.2.0%clang@=14.0.0-rocm5.2.0-mixed amdgpu_target=auto build_system=cmake build_type=Release generator=make arch=linux-sles15-zen3
 module load rocthrust/5.2.0-clang-14.0.0-rocm5.2.0-mixed-3upg7wr
 # ginkgo@=1.5.0.glu_experimental%clang@=14.0.0-rocm5.2.0-mixed~cuda~develtools~full_optimizations~hwloc~ipo~mpi~openmp+rocm+shared~sycl amdgpu_target=gfx90a build_system=cmake build_type=Release generator=make patches=ba0956e arch=linux-sles15-zen3
-module load ginkgo/1.5.0.glu_experimental-clang-14.0.0-rocm5.2.0-mixed-fo65t4m
+module load ginkgo/1.5.0.glu_experimental-clang-14.0.0-rocm5.2.0-mixed-6bembpq
+# gmake@=4.4.1%gcc@=12.2.0-mixed~guile build_system=generic arch=linux-sles15-zen3
```

Comment:
We need Ginkgo >= 1.7. I think `glu_experimental` branch will be deleted soon anyways.

Reply:
I am not aware that ginkgo@1.7 interface to HiOp has been update.

Reply:
It has not. However, `glu_experimental` is unsupported Ginkgo branch incompatible with mainstream Ginkgo, so it is not clear which solution is worse.